ingest_graph_validator/actions/ingest_validator_action.py

```diff
@@ -80,14 +95,19 @@ def handle_message(self, message):
 
             validation_result = ValidationHandler(subid, self._graph, self._test_path).run()
 
-            if validation_result is not None:
+            if validation_result is not None:   
                 self._logger.info(f"validation finished for {subid}")
 
                 if validation_result["valid"]:
                     self._ingest_api.put(f'{submission_url}/graphValidEvent', data=None)
                 else:
-                    self._ingest_api.put(f'{submission_url}/graphInvalidEvent', data=validation_result["failures"])
+                    self._ingest_api.put(f'{submission_url}/graphInvalidEvent', data=None)
+
+                    for failure in validation_result["failures"]:
+                        for entity in failure['affectedEntities']:
+                            self.__patch_entity(failure['message'], entity['types'][0], entity['uuid'])
 
+                self._logger.info(f'Finished validating {subid}.    ')
```

Comment:
could rename subid to be sub_uuid or submission_uuid

Reply:
Done!